### ISSUE TITLE
[Feature request] Ability to have custom columns, rename of columns

### ISSUE THREAD

**Issue by gudaol** [mkloubert/vscode-kanban#14]

**Comment by mkloubert** [comment]
@gudaol 

**I released a new version 1.20.0**, where you can define custom column names now.

```json
{
    "kanban": {
        "columns": {
            "done": "Finished",
            "inProgress": "Work",
            "testing": "Currently Testing",
            "todo": "Ideas and TODOs"
        }
    }
}
```

**Comment by gudaol** [comment]
Ooh, nice! Thank you!
